installer/PowerToysSetup/Product.wxs

```diff
@@ -139,7 +143,19 @@
+  <?else?>
+    <?define DotnetRuntimeWPFFiles=Accessibility.dll;DirectWriteForwarder.dll;Microsoft.VisualBasic.Forms.dll;Microsoft.Win32.Registry.AccessControl.dll;Microsoft.Win32.SystemEvents.dll;PenImc_cor3.dll;PresentationCore.dll;PresentationFramework-SystemCore.dll;PresentationFramework-SystemData.dll;PresentationFramework-SystemDrawing.dll;PresentationFramework-SystemXml.dll;PresentationFramework-SystemXmlLinq.dll;PresentationFramework.Aero.dll;PresentationFramework.Aero2.dll;PresentationFramework.AeroLite.dll;PresentationFramework.Classic.dll;PresentationFramework.dll;PresentationFramework.Luna.dll;PresentationFramework.Royale.dll;PresentationNative_cor3.dll;PresentationUI.dll;ReachFramework.dll;System.CodeDom.dll;System.Configuration.ConfigurationManager.dll;System.Design.dll;System.Diagnostics.EventLog.dll;System.Diagnostics.EventLog.Messages.dll;System.Diagnostics.PerformanceCounter.dll;System.DirectoryServices.dll;System.Drawing.Common.dll;System.Drawing.Design.dll;System.IO.Packaging.dll;System.Security.Cryptography.Pkcs.dll;System.Security.Cryptography.ProtectedData.dll;System.Security.Cryptography.Xml.dll;System.Security.Permissions.dll;System.Threading.AccessControl.dll;System.Windows.Controls.Ribbon.dll;System.Windows.Extensions.dll;System.Windows.Forms.Design.dll;System.Windows.Forms.Design.Editors.dll;System.Windows.Forms.dll;System.Windows.Forms.Primitives.dll;System.Windows.Input.Manipulations.dll;System.Windows.Presentation.dll;System.Xaml.dll;UIAutomationClient.dll;UIAutomationClientSideProviders.dll;UIAutomationProvider.dll;UIAutomationTypes.dll;vcruntime140_cor3.dll;WindowsFormsIntegration.dll;wpfgfx_cor3.dll?>
+  <?endif?>
```

Comment:
These big ifs are likely hard to maintain. What is the difference between arm64 and x64 here? Perhaps just  those can be "if"-ed instead and that new define be added on the components that need it.

Reply:
I'll see how many files differ there. Anyway, I have an idea of automating this somehow, instead of doing it like this.. But I'll rather leave that as a follow-up PR